test/letters/containers/VeteranBenefitSummaryLetter.unit.spec.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import { VeteranBenefitSummaryLetter } from '../../../src/js/letters/containers/VeteranBenefitSummaryLetter';
+
+const defaultProps = {
+  benefitSummaryOptions: {
+    benefitInfo: {
+      awardEffectiveDate: '1965-01-01T05:00:00.000+00:00',
+      hasAdaptedHousing: true,
+      hasChapter35Eligibility: false
+    },
+    serviceInfo: [
+      {
+        branch: 'ARMY',
+        characterOfService: 'UNCHARACTERIZED',
+        enteredDate: '1965-01-01T05:00:00.000+00:00',
+        releasedDate: '1972-10-01T04:00:00.000+00:00'
+      }
+    ]
+  },
+  optionsAvailable: true,
+  optionsToInclude: {
+  }
+};
+
+describe('<VeteranBenefitSummaryLetter>', () => {
+  it('should render', () => {
+    const tree = SkinDeep.shallowRender(<VeteranBenefitSummaryLetter {...defaultProps}/>);
+    const vdom = tree.getRenderOutput();
+    expect(vdom).to.exist;
+  });
+
+  it('should show benefit info options', () => {
+    const tree = SkinDeep.shallowRender(<VeteranBenefitSummaryLetter {...defaultProps}/>);
+    expect(tree.subTree('hasAdaptedHousing')).to.exist;
+  });
+
+  it('should show service info options', () => {
```

Comment:
In writing this, I began to wonder if we should be handling the case of an empty military service list (although that shouldn't be possible/make sense). I think https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/schemas/letter_beneficiary.json#L54 should be updated to require a non-empty array. I'll update the above issue.